java/test/org/openqa/selenium/environment/webserver/CookieHandler.java

```diff
@@ -191,4 +191,19 @@ private Cookie parse(String cookieString) {
 
     return builder.build();
   }
+  private String escapeCookieValue(String value) {
+    if (value == null || value.isEmpty()) {
+      return "";
+    }
+
+    return value.replace("\\", "\\\\")
+      .replace("\"", "\\\"")
```

Comment:
Please use StringBuilder. This will create a new string each time replace is called.

Reply:
Will do so!